venv/pysnes/cpu.py

```diff
@@ -1987,14 +2138,28 @@ def sub_twos_complement(self, value, arg, is8BitMode):
 
 
     # compute twos complement by hand.
-    def add_twos_complement(self, value, arg, is8BitMode):
+    def add_twos_complement(self, value, arg, is8BitMode, affectCVFlags=False):
```

Comment:
I did that so it could still be used by inc. Since the flag is false by default, we don't have to change the inc opcodes at all.

Reply:
Yep.Thats the way to extend functions in Python. However, this function will not be used by inc in the future because it makes no sense to use a function for one add. Will you repair the inc impl and test or should I do it?

Reply:
Ah I see. I'll repair inc and the tests and also remove the affectCVFlags argument.